webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts

```diff
@@ -267,7 +267,7 @@ export class AdministrationScreenService {
       return true;
     }
 
-    return this.administrationItemService.canDeActivate(screen, toScreen, this.isConfigurationMode, screen.item !== toScreen?.item);
+    return await this.administrationItemService.canDeActivate(screen, toScreen, this.isConfigurationMode, screen.item !== toScreen?.item);
```

Comment:
why do we need return await here?

Reply:
As I can see that function returns `boolen | Promise<boolean>`, not only `Promise<boolean>`
It's better to fix return type